get/config.yaml

```diff
@@ -4,7 +4,7 @@ slug: get
 image: "ghcr.io/hacs/{arch}-addon-get"
 description: The easiest way to get HACS for Home Assistant
 startup: once
-boot: manual
+boot: manual_only
```

Comment:
Do you know of any (official?) add-ons that have had a similar change?
Im curious about what will happen to those who have set this toggle previously.